tests/release/e2e-test-default-bundle.go

```diff
@@ -0,0 +1,240 @@
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates release plan admission in the managed namespace.", func() {
+			_, err := framework.ReleaseController.CreateReleasePlanAdmission(targetReleasePlanAdmissionName, devNamespace, applicationNameDefault, managedNamespace, "", "", releaseStrategyDefaultName)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates EnterpriseContractPolicy in the managed namespace.", func(ctx SpecContext) {
+			_, err := framework.TektonController.CreateEnterpriseContractPolicy(releaseStrategyPolicyDefault, managedNamespace, defaultEcPolicy)
+			Expect(err).NotTo(HaveOccurred())
+		}, SpecTimeout(EnterpriseContractPolicyTimeout))
+
+		It("creates pvc in the managed namespace.", func() {
+			_, err := framework.TektonController.CreatePVCInAccessMode(releasePvcName, managedNamespace, corev1.ReadWriteOnce)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates sevice account in the manged namespace.", func() {
+			_, err := framework.CommonController.CreateServiceAccount(releaseStrategyServiceAccountDefault, managedNamespace, managednamespaceSecret)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates role in the managed namespace.", func() {
+			_, err := framework.CommonController.CreateRole(roleName, managedNamespace, roleRules)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates Role binding with a given name in managednamespace.", func() {
+			_, err := framework.CommonController.CreateRoleBinding("role-relase-service-account-binding", managedNamespace, "ServiceAccount", roleName, "Role", "role-m6-service-account", "rbac.authorization.k8s.io")
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates application with the given name in dev namespace.", func() {
+			_, err := framework.HasController.CreateHasApplication(applicationNameDefault, devNamespace)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates component with the given component name in dev namespace.", func() {
+			_, err := framework.HasController.CreateComponent(applicationNameDefault, componentName, devNamespace, gitSourceComponentUrl, "", containerImageUrl, "", "", false)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+	})
+
+	var _ = Describe("Post-release verification", func() {
+
+		It("verifies that in dev namespace will be created a PipelineRun.", func() {
+			Eventually(func() bool {
+				prList, err := framework.TektonController.ListAllPipelineRuns(devNamespace)
+				if err != nil || prList == nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return strings.Contains(prList.Items[0].Name, componentName)
+			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies that the PipelineRun in dev namespace succeeded.", func() {
```

Comment:
Is this the Build PipelineRun or a Release pipelinerun?

Reply:
yes, 
this is the Build one